app/lib/dfe/bigquery/table.rb

```diff
@@ -0,0 +1,27 @@
+module DfE
+  module Bigquery
+    class Table
+      def self.client
+        require 'google/cloud/bigquery'
+
+        config = ::DfE::Bigquery.config
+
+        missing_config = %i[
```

Comment:
I need to have a think about this. Don't think you don't want to check in the client, but in the configurator if the settings are present. You will be caling the client in multiple places, this doesn't seem right.

Reply:
The main thing I want to avoid if the app raises an error if the env var is not there. (because we don't have the env values for alle envs yet)

Reply:
Yeah, I saw that. But is it necessary to check if the configs exist? It's not public facing, and once they get set, it'll be fine, we don't have to verify this constantly. Maybe I'm missing something.

Reply:
If I memoize then it would only call once.

Reply:
If you do want to have this functionality, I think what would work best would be to move the `missing_config` bit to `bigquery.rb`(or however you want to call it) and put it in a class method `self.missing_config?` or I'd reverse it and call it `self.valid?` and then raise here on `unless config.valid?` This way you can also remove the duplication of defining the `configurables` and `missing_config`. Thoughts?

Reply:
Done!